examples/cqt2midi.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import division
+"""
+Utility function for converting an audio file
+to a MIDI file. Note that this method is nowhere close
+to the state-of-the-art in automatic music transcription.
+This just serves as a fun example for rough
+transcription which can be expanded on for anyone motivated.
+"""
+import pretty_midi
+import numpy as np
+import librosa
+import sys
+import argparse
+
+
+def cqt_to_piano_roll(cqt, min_midi, max_midi, threshold):
+    '''Convert a CQT spectrogram into a piano roll representation by
+     thresholding scaled magnitudes.
+
+    Parameters
+    ----------
+    cqt : np.ndarray, shape=(max_midi-min_midi,frames), dtype=complex64
+        CQT spectrogram of audio.
+    min_midi : int
+        Minimum MIDI note to transcribe.
+    max_midi : int
+        Maximum MIDI note to transcribe.
+    threshold : int
+        Threshold value to activate note on event, 0-127
+
+    Returns
+    -------
+    piano_roll : np.ndarray, shape=(128,frames), dtype=int
+        Piano roll representation on audio.
+
+    '''
+    piano_roll = np.abs(cqt)
+    piano_roll = np.digitize(piano_roll, np.linspace(piano_roll.min(),
+                                                     piano_roll.max(), 127))
+    piano_roll[piano_roll < threshold] = 0
+    piano_roll = np.pad(piano_roll,
+                        [(128-max_midi, min_midi), (0, 0)], 'constant')
+    return piano_roll
+
+
+def piano_roll_to_pretty_midi(piano_roll, fs=100, program=0):
```

Comment:
Nitpick, but can you switch the order of these two functions? It's `piano_roll_to_pretty_midi` that you are showcasing here, so makes sense to put it first.